rest-api-spec/src/main/resources/schema.json

```diff
@@ -130,7 +130,7 @@
             "additionalProperties": false,
             "properties": {
                 "url": {
-                    "type": "string",
+                    "type": ["string", "null"],
```

Comment:
Can the use of `null` be constrained only to APIs that have stability `beta` or `experimental`? This would align with APIs that are under development, whilst still providing benefit of having links for stable APIs

Reply:
I made the null conditional. So that only `stable` requires a string, the others now allow null.  https://github.com/elastic/elasticsearch/pull/55749/commits/0b379774dc73b545c9e1f3be89f89c8e3de79bf8

As you pointed out below, this does indeed cause a validation error on the two `ml` specs below we will need to sort out first.